### DISCUSSION TITLE
Does esapi-java-legacy support jDK17

### DISCUSSION THREAD

**Discussion by JerryDevis** [ESAPI/esapi-java-legacy#806] [Q&A]
Does esapi-java-legacy support jDK17?

**Comment by kwwall** [comment]
If you are using it with the standard Java Servlet API (e.g., the "javax.servlet" package name space), I think it ought to work. If you are using Java 17 with the Jakarta Servlet API (package name space of "jakarta.servlet"), then it one. We have a PR for that that I've tested and hope to get into this new release that I'm working on. See https://github.com/ESAPI/esapi-java-legacy/discussions/768 for more details.
